### PR TITLE
pre_bump_release, pre_add_labels_in_df: relocate setting release variable

### PR DESCRIPTION
**Pull request by mlangsdorf**
Move setting the release environment variable out of pre_bump_release that is only called from the orchestrator and into pre_add_labels_in_df that is called by workers and orchestrator.

Signed-off-by: Mark Langsdorf <mlangsdo@redhat.com>

Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request includes link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

File: atomic_reactor/schemas/container.json [hunk 1]
```diff
@@ -165,15 +165,6 @@
...
     },
-    "set_release_env": {
-      "type": "string",
-      "description": "Name of an environment variable to set in the Dockerfile file with the release number",
-      "examples": [
-        "CONTAINER_RELEASE_NVR",
-        "IMAGE_RELEASE",
-        "OS_BUILD"
-      ]
-    },
```
Comment (twaugh) [inline atomic_reactor/schemas/container.json]: Let's not revert this change!

Comment (mlangsdorf) [reply atomic_reactor/schemas/container.json]: Fixed.

**Comment by mlangsdorf** [comment]
added to release notes under Bug Fixes:
- release environment variables are set for both worker and orchestrator builds
